patch.sh

```diff
@@ -71,8 +71,9 @@ declare -A object_list=(
     ["415.25"]='libnvcuvid.so'
 )
 
+NVIDIA_SMI="$(which nvidia-smi)"
```

Comment:
Please use `command -v` built-in in order to locate command. See rationale here: https://www.shellcheck.net/wiki/SC2230